### PR TITLE
Added labels to mini floating action buttons

### PR DESCRIPTION
**Pull request by molly34762**
- Added labels next to mini floating action buttons
- changes background tint of add button upon clicking

### CODE REVIEW

File: app/src/main/res/layout/mini_manual_add_device_fab.xml [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
-    android:layout_height="wrap_content"
-    android:src="@drawable/ic_baseline_insert_drive_file_24"
-    android:visibility="invisible"
-    app:fabSize="mini"
-    style="@style/Widget.Carebase.MiniFloatingActionButton" />
\ No newline at end of file
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:visibility="gone"
+    android:orientation="vertical">
```
Comment (sak6lab) [inline app/src/main/res/layout/mini_manual_add_device_fab.xml:8]: Change this to `horizontal`

File: app/src/main/res/layout/mini_manual_add_device_fab.xml [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
+
+    <TextView
+        android:id="@+id/manual_add_device_fab_label"
```
Comment (sak6lab) [inline app/src/main/res/layout/mini_manual_add_device_fab.xml:11]: - Add 4dp padding and 4dp horizontal padding
- Add rounded corners by setting a white rounded corner background
- Add elevation of 6dp to match floating action buttons

**Review by sak6lab — CHANGES_REQUESTED** [review]
The labels are not that apparent as there is no background for the labels. Also change title of this PR to something more descriptive like 

> Added labels to mini floating action buttons

Other than that you should be good

**Comment by molly34762** [comment]
@sak6lab I made some changes to the UI, can you take a quick look to confirm? Thanks!

**Comment by molly34762** [comment]
@sak6lab I have changed the ui of the labels, however I am not sure why the onclick for the scanning and adding device button doesn't work.

**Comment by sak6lab** [comment]
Screen Shot 2021-02-06 at 12 33 23 PM

- gave labels uniform width to make it uniform
- cleaned up animations
- made actual mini button unclickable to avoid the issue where mini buttons where clickable and would not perform correctly

**Review by sak6lab — APPROVED** [review]

**Pull request merged**